modules/lightmapper_rd/lightmapper_rd.cpp

```diff
@@ -356,7 +356,7 @@ void LightmapperRD::_create_acceleration_structures(RenderingDevice *rd, Size2i
 	bounds = AABB();
 
 	for (int m_i = 0; m_i < mesh_instances.size(); m_i++) {
-		if (p_step_function) {
+		if (p_step_function && mesh_instances.size() > 0) {
```

Comment:
This is known to be greater than zero, as we are iterating through it

Reply:
The compiler is stupid and I'll remove it.